build/gen_compile_db.py

```diff
@@ -1,52 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
I think we want to keep this file. It's used by cmake to postprocess compile_commands.json